package.json

```diff
@@ -24,9 +24,6 @@
   ],
   "author": "Chris Sullivan",
   "license": "Unlicense",
-  "peerDependencies": {
-    "react": "^15.6.1"
-  },
```

Comment:
There are a couple cases here that I'm worried about. 

1. We limit individuals on an earlier version of React (which wouldn't be a problem if we minor versioned the change)
2. The JSX pragma for createClass would not be found and the spinner would never create a dom element.

I'm testing in the example to see if the 2nd concern is warranted. 
2.

Reply:
Tested this and everything seems to work. I might have misunderstood the docs for peerDependency. I'll merge this and do a patch version.